basics/src/main/java/basics/ConvertibleCar.java

```diff
@@ -0,0 +1,35 @@
+package basics;
+
+public class ConvertibleCar extends Car {
+	
+	private boolean TopMotor;
+	
+	public ConvertibleCar() {
+		this.model = null;
+		this.odometer = 0;
+		engine = new SmallEngine();
+		TopMotor = false;
+	}
+	
+	public ConvertibleCar(String model) {
+		this.model = model;
+		this.odometer = 0;
+		engine = new SmallEngine();
+		TopMotor = false;
+	}
+	
+	public void SetTopMotor(String Requirement) {
+		if (Requirement == "Raise") {
```

Comment:
boolean would work better here since there are only two options for the top (up/down)